packages/b/bgfx/rules/shaders.lua

```diff
@@ -73,8 +61,8 @@ rule("shaders")
         }
         local bgfx_default_profiles = {
             windows = {
-                vertex = {dx9 = "s_3_0", dx11 = "s_5_0", glsl = "120"},
-                fragment = {dx9 = "s_3_0", dx11 = "s_5_0", glsl = "120"},
```

Comment:
Why removing dx9 support? In xmake-repo old versions of bgfx are also available

Reply:
I didn't remove the dx9 support, I only removed it from the default profiles. When the project doesn't specify the bgfx version; by default, it uses the latest (currently 8752) which doesn't have the RendererType::Direct3D9.